polymorphic_value.h

```diff
@@ -134,8 +132,7 @@ class delegating_control_block : public control_block<T> {
       : delegate_(std::move(b)) {}
 
   ISOCPP_P0201_CONSTEXPR_CXX20
-      std::unique_ptr<control_block<T>, control_block_deleter>
-      clone()
+  std::unique_ptr<control_block<T>, control_block_deleter> clone()
```

Comment:
Code coverage failure here is not helpful.  This is just formatting so no coverage changes should be expected.